src/Microsoft.OpenApi/Extensions/StringExtensions.cs

```diff
@@ -1,7 +1,8 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT license. 
+// Licensed under the MIT license.
```

Comment:
```suggestion
// Licensed under the MIT license. 
```
Fake update to trigger build